actions/admin/settings/180.dkim.php

```diff
@@ -88,10 +88,10 @@
 					'default' => '0',
 					'option_mode' => 'one',
 					'option_options' => array(
-						'0' => 'All',
```

Comment:
completely abusing existing setting, makes no sense

Reply:
I do not find this line in my code